### PR TITLE
[gui] Fixes #1654 - Remove owner parameter when unzipping dist files

### LINKED ISSUE #1654: Use zip to zip files fails on linux
Hi,

Latest changes introduced on: 
https://github.com/skycoin/skycoin/commit/1abddb4409274e30bfb75f75091b8cc1e65284d5

Are throwing error:
```
------------------------------
Compressing standalone release
Removing old mdl-0.23.0-bin-osx-darwin-x64.zip
Zipping mdl-0.23.0-bin-osx-darwin-x64.zip
Zipping mdl-0.23.0-bin-win-x64.zip

zip error: Invalid command arguments (long option 'owner' not supported)
```

Any ideas what might be wrong ?

### PR DESCRIPTION
**Pull request by stdevHan**
Fixes #1654

Changes:
-Remove parameters `--owner=0 --group=0` the  `electron/compress-standalone-release.sh`, because modern zip versions do not use those parameters.

Does this change need to mentioned in CHANGELOG.md?

### CODE REVIEW

File: electron/compress-standalone-release.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [ -e "$WIN64_STL" ]; then
     fi
     echo "Zipping $WIN64_STL_ZIP"
     if [[ "$OSTYPE" == "linux"* ]]; then
-        zip -r --quiet "$WIN64_STL_ZIP" --owner=0 --group=0 "$WIN64_STL"
+        zip -r --quiet "$WIN64_STL_ZIP"  "$WIN64_STL"
```
Comment (olemis) [inline electron/compress-standalone-release.sh]: @stdevHan gotta set permissions with `chown` .

File: electron/compress-standalone-release.sh [hunk 2]
```diff
@@ -57,7 +57,7 @@ if [ -e "$WIN32_STL" ]; then
     fi
     echo "Zipping $WIN32_STL_ZIP"
     if [[ "$OSTYPE" == "linux"* ]]; then
-        zip -r --quiet "$WIN32_STL_ZIP" --owner=0 --group=0 "$WIN32_STL"
+        zip -r --quiet "$WIN32_STL_ZIP" "$WIN32_STL"
```
Comment (olemis) [inline electron/compress-standalone-release.sh]: @stdevHan gotta set permissions with `chown` .

File: electron/compress-standalone-release.sh [hunk 3]
```diff
@@ -74,7 +74,7 @@ if [ -e "$LNX64_STL" ]; then
     fi
     echo "Zipping $LNX64_STL_ZIP"
     if [[ "$OSTYPE" == "linux"* ]]; then
-        tar czf "$LNX64_STL_ZIP" --owner=0 --group=0 "$LNX64_STL"
+        tar czf "$LNX64_STL_ZIP"  "$LNX64_STL"
```
Comment (olemis) [inline electron/compress-standalone-release.sh]: @stdevHan gotta set permissions with `chown` .

**Comment by gz-c** [comment]
The owner and group need to be wiped to remove artifacts indicating which user built the software

**Comment by olemis** [comment]
@thomasmodeneis can you please check last changes in this PR work for you ?

**Pull request merged**